File: PageantManager.Web/src/app/shared/models/garment-measurement-type.ts
import { MeasurementType } from './measurement-type';

export class GarmentMeasurementType {
  garmentMeasurementTypeId: number;
  garmentTypeId: number;
  measurementTypeId: number;
  measurementType: MeasurementType;

  public constructor(init?: Partial<GarmentMeasurementType>) {
    Object.assign(this, init);
  }
}

File: PageantManager.Core/Entities/GarmentMeasurementType.cs
namespace PageantManager.Core.Entities
{
    public class GarmentMeasurementType : Entity
    {
        public int GarmentMeasurementTypeId { get; set; }

        public int GarmentTypeId { get; set; }
        public int MeasurementTypeId { get; set; }

        public GarmentType GarmentType { get; set; }
        public MeasurementType MeasurementType { get; set; }
    }
}